### PR TITLE
Add rule to allow multicast DNS

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: manifests/rules/mdns.pp [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#
+# @summary allow incoming multicast DNS
+#
+class nftables::rules::mdns {
+  nftables::mdns {
+    'default_in-multicast':
+      content => 'ip daddr 224.0.0.251 accept',
```
Comment (kenyon) [inline manifests/rules/mdns.pp:7]: Formatting seemed odd

```suggestion
  nftables::mdns { 'default_in-multicast':
    content => 'ip daddr 224.0.0.251 accept',
```

Comment (kenyon) [reply manifests/rules/mdns.pp]: Then I saw that all of the rules are formatted like that 😢

**Review by kenyon — APPROVED** [review]
Legacy IP only though 😉

**Comment by bastelfreak** [comment]
spotify doesn't know IPv6 :(

**Review by kenyon — APPROVED** [review]

**Pull request merged**